docs/source/_json/content_get_versioned.req

```diff
@@ -0,0 +1,3 @@
+GET /plone/folder/my-document?version_id=0 HTTP/1.1
```

Comment:
@jaroel it might be more consistent with our current API design if we would retrieve a specific version with "my-document/@history/<VERSION_ID>". The hypermedia links would also be a bit nicer because we would provide a collection-like API response on the /@history endpoint:
```

[
    {
        "@id": "http://localhost:55001/plone/front-page/@history/0",
        "action": null,
    },
    {
        "@id": "http://localhost:55001/plone/front-page/@history/1",
        "action": "Edited",
    }
]

```

Though, this might lead to a problem with our initial revision that does not have a version id.